manifestinteractive/jqvmap
# Fixes #238 Pins Positioned Incorrectly

#### What does this PR do ?

Fixes #238 Pins Positioned Incorrectly.

#### How should this be tested ?

Should fix: http://codepen.io/anon/pen/QEbWvQ
Sorry I didn't have time to add any tests in the build.

#### What are the relevant issues ?

#238 

#### This Pull Request includes:

- [ ] Bug Fix with passing `npm test`
- [ ] New Feature with passing `npm test`
- [ ] Code Improvement with passing `npm test`
- [ ] New Map File & Example HTML

#### What gif best describes how you feel about this work ?

```
¯\_(ツ)_/¯
```
